build.gradle

```diff
@@ -1,20 +1,72 @@
-allprojects {
-    repositories {
-        mavenCentral()
+apply plugin: 'idea'
+apply plugin: 'java'
+apply from: 'external-dependencies.gradle'
+apply from: 'idea-config.gradle'
+
+task wrapper(type: Wrapper) {
+    gradleVersion = '2.6'
+}
+
+group 'mint'
+
+repositories {
+    mavenCentral()
+}
+
+//noinspection GroovyAssignabilityCheck
+dependencies {
+    compile dropwizard, apacheCommonsCodec
+
+    compile(postgresClient) {
+        exclude group: 'org.slf4j'
     }
-    apply plugin: 'idea'
-    apply from: "${rootProject.projectDir}/external-dependencies.gradle"
+
+
+    testCompile junit, mockito, dropwizardTest
 }
 
-subprojects {
-    apply plugin: 'java'
-    sourceCompatibility = "1.8"
-    targetCompatibility = "1.8"
+
+jar.baseName = "mint"
+
+jar {
+    doFirst {
+        from {
+            configurations.compile.collect {
+                it.isDirectory() ? it : zipTree(it)
+            }
+        } {
+            exclude "META-INF/*.SF"
+            exclude "META-INF/*.DSA"
+            exclude "META-INF/*.RSA"
+        }
+
+        manifest {
+            attributes(
+                    "Main-Class": "uk.gov.MintApplication",
+                    "Class-Path": configurations.runtime.findAll { !it.directory }.collect { it.name }.join(' ')
+            )
+        }
+    }
 }
 
-apply from: 'idea-config.gradle'
+assemble {
+    doLast {
+        new File("deploy/mint.jar").bytes = new File("build/libs/mint.jar").bytes
+        createDeployableBundle.execute()
+    }
+}
 
-task wrapper(type: Wrapper) {
```

Comment:
I think we still need wrapper task. It is required to regenerate contents of gradle directory when update the gradle version.